hexnumber/hexnum.py

```diff
@@ -0,0 +1,92 @@
+import sys
```

Comment:
Your pull request: HEXNUMBER-1022 Added hexnumber.py


Your commit: HEXNUMBER-1022 Added hexnumber.py

Your commit: HEXNUMBER-1022 Refactored hexnumber.py
Action of your comment must be from ['Added', 'Deleted']
Your commit: HEXNUMBER-1022 Added testhexnumber.py

Your commit: HEXNUMBER-1022 Refactored hexnumber.py
Action of your comment must be from ['Added', 'Deleted']
Your commit: HEXNUMBER-1022 Refactored testhexnumber.py
Action of your comment must be from ['Added', 'Deleted']
Your commit: HEXNUMBER-1022 Refactored hexnum.py
Action of your comment must be from ['Added', 'Deleted']
Your commit: HEXNUMBER-1022 Refactored hexnum.py and tests
Action of your comment must be from ['Added', 'Deleted']